Docs/source/running_cpp/parameters.rst

```diff
@@ -1738,7 +1738,7 @@ Lookup tables store pre-computed values for functions used by the QED modules.
     is quite low).
 
     * ``generate``: a new table is generated. This option requires Boost math library
-      (version >= 1.67) and to compile with QED_TABLE_GEN=TRUE. All
+      (version >= 1.67) and to compile with ``QED_TABLE_GEN=TRUE``. All
```

Comment:
```suggestion
      (version >= 1.66) and to compile with ``QED_TABLE_GEN=TRUE``. All
```
With a small change that I made few days ago it works nicely with Boost 1.66 (it's the version available on Summit)!